app/views/user_mailer/email_confirmation_instructions.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="lead">
```

Comment:
the old template had `<p class="lead">` -- was there a reason to change to a `<div>`?

Reply:
The original text was in one paragraph with this class, which created a certain amount of space between the text and the big button.  As I broke the text into 2 P tags, I wrapped them in a div to preserve that space.

Reply:
This commit also broke a bunch of tests related to the original email template, which I now understand has more use cases besides unconfirmed emails.  I could restore the original template then add logic to display certain parts based on `user.confirmed?`, something like:
```
<% if user.confirmed? %>
  <!-- paragraph 1 -->
<% else %>
  <!-- paragraph 1>
  <!-- paragraph 2>
<% end %>

<!-- big-button link inside table -->

<% if user.confirmed? %>
  <!-- confirm email (create user) link -->
<% else %>
  <!-- paragraph with confirm email (create user) link -->
<% end %>
```
but we want to avoid logic in ERBs where possible, right?

So in the next commit, I

- add another `UserMailer` method and associated ERB for unconfirmed email
- update the logic to call the original or unconfirmed email instructions method in `UserMailer` based on `user.confirmed?`

I'm open to a different or more elegant solution, but at least it's straightforward IMHO

Reply:
I'm still seeing way too many test failures.  I believe the root is in how I'm determining if the email is not confirmed.  Comments are welcome, but another commit is coming.

Reply:
After further research, it looks like `user.confirme?` is doing what I expect, so my issue is elsewhere.  This PR is getting sloppy, so I'm going to close and start again.  Sorry for any inconveniene.

Reply:
If there are essentially 2 emails, we could just have two templates? And branch in the calling code

Reply:
That's the plan, which I tried to outline above